app/views/taxon_framework_relationships/_show_taxon_framework_relationship.html.haml

```diff
@@ -0,0 +1,36 @@
+:ruby
+  taxon_framework_relationship ||= show_taxon_framework_relationship[:taxon_framework_relationship]
+  taxon ||= show_taxon_framework_relationship[:taxon]
+%p
+  = "#{ t(:relationship) }:"
```

Comment:
This is also not great form for translation. Better to use something like `relationship_labeled: "Relationship: %{relationship}"`

Reply:
done